logging/README.md

```diff
@@ -0,0 +1,228 @@
+    <artifactId>quarkus-google-cloud-logging</artifactId>
+    <version>${googleCloudServicesVersion}</version>
+</dependency>
+```
+
+## How-to: Log
+When configured, all that is needed is to log data as you wish. You can use `JUL`, `jboss-logging` or `Slf4j`, or even GCP logging directly. You can also 
+inject the logger to use, so for example: 
+
+```java
+package mypackage;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.jboss.logging.Logger;
+
+@Path("/logging")
+public class LoggingResource {
+
+    @Inject
+    Logger logger;
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String tryLog() {
+        logger.info("Greeting the World");
+        return "Hello World!";
+    }
+}
+```
+
+##  Configuration
+
+### Basic
+Currently the logging system uses a single GCP Operations log and this is a mandatory configuration parameter. 
+
+* `quarkus.google.cloud.logging.default-log`
+* `quarkus.google.cloud.project-id`
+
+If you which to disable the logging to GCP you can do so: 
+
+* `quarkus.google.cloud.logging.enabled=false`
+
+This is useful for testing, not to write to Google Operations during development: 
+
+* `%test.quarkus.google.cloud.logging.enabled=false`
+
+### Formatting
+This logging system can either use a standard text based format, Elastic Common Schema, or a completely custom schema.    
+
+* `quarkus.google.cloud.logging.format=[TEXT|JSON]`
+
+#### Structured Logging
+By setting the logging `format` to `JSON` this library will default to ECS format. As loca record parameters and MDC values are not
+official part of the ECS specification, these default to `parameters`  and `mdc` respectively, but this can be configured (see below).
+
+* `quarkus.google.cloud.logging.structured.stack-trace.included=[true|false]` - Should stack traces be included? Default to `true`
+* `quarkus.google.cloud.logging.structured.stack-trace.rendering=[array|string]` - Render stack traces as a strings or an arrays of objects, defaults to `string`
+* `quarkus.google.cloud.logging.structured.stack-trace.element-rendering=[string|object]` - How to render each stack trace element, defaults to `string`
+* `quarkus.google.cloud.logging.structured.parameters.included=[true|false]` - Should log record parameters be included? Defaults to `true`
+* `quarkus.google.cloud.logging.structured.parameters.field-name` - This is the field name to use in ESC for parameters, defaults to `parameters`
+* `quarkus.google.cloud.logging.structured.mdc.included=[true|false]` - Should the MDC values be included? Defaults to `true`
+* `quarkus.google.cloud.logging.structured.mdc.field-name` - This is the field name to use in ESC for MDC values, defaults to `mdc`
+
+You can use a completely custom structured format by binding a `JsonFormatter` to the CDI context. For example, if you want to filter the 
+included parameters, you could do this: 
```

Comment:
Is using a custom JSON formatter really useful and something someone will do ?
If it's a very rare use case, I'll suggest to drop it as most user will use the EcsJson formatteur and there is already a lot of custimization possible.

Reply:
Good suggestion, will fix.